frontend/src/design-system/visibility.css

```
@@ -0,0 +1,81 @@
+
+/* Withheld: the one state about the reader, drawn as the caution it is. The
+   tint composites over its own --bgElevated so its contrast does not depend on
+   which surface a caller put it on — the same reason .badge-warn does. */
+.visibility-withheld {
+  background: linear-gradient(var(--warnBg), var(--warnBg)), var(--bgElevated);
+  border-color: transparent;
+  color: var(--warn);
+}
+
+.visibility-withheld > svg {
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)